### PR TITLE
Adding the Mobile Network Performance Dashboard

### PR DESCRIPTION
**Pull request by siddy15**
This is a comprehensive dashboard that provides the visibility about the mobile application network performance, their frequency, affect versions etc.
There is no dashboard available for mobile network performance analysis and there is lot of manual effort each time when this type of dashboard is needed.

### CODE REVIEW

File: dashboards/mobile-network-performance/network-performance.json [hunk 1]
```diff
@@ -0,0 +1,2108 @@
+{
+  "name": "Mobile Network Performance",
+  "description": null,
+  "permissions": "PRIVATE",
```
Comment (caylahamann) [inline dashboards/mobile-network-performance/network-performance.json:4]: permissions field shouldn't be used 
```suggestion
```

**Review by caylahamann — COMMENTED** [review]
some minor changes needed!

File: dashboards/mobile-network-performance/network-performance.json [hunk 2]
```diff
@@ -0,0 +1,2108 @@
+{
+  "name": "Mobile Network Performance",
+  "description": null,
```
[the remark this answers is not in the document]
Comment (siddy15) [reply dashboards/mobile-network-performance/network-performance.json]: Description field updated.

File: install/mobile-network-performance/install.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+id: mobile-network-performance-install
```
Comment (caylahamann) [inline install/mobile-network-performance/install.yml:1]: Sorry didn't catch this on first review, but we're actually deprecating the use of install plans in favor of data sources. 
Here's a guide on the conversion: 
https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#install-plans-deprecated

**Review by zstix — APPROVED** [review]

**Pull request merged**